my-code/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "react-webpack-babel",
+  "version": "0.0.4",
+  "description": "React Webpack Babel Starter Kit",
+  "main": "''",
+  "engines": {
+    "node": "10.12.0"
+  },
+  "scripts": {
+    "clean": "shx rm -rf build",
+    "webpack": "webpack --config config/webpack.config.production.js",
+    "sass": "node-sass src/styles/index.scss build/index.css --output-style compressed",
+    "build": "npm run clean && npm run sass && npm run webpack",
+    "lint": "eslint src config",
+    "dev": "webpack-dev-server --config config/webpack.config.dev.js",
+    "start": "serve -s build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ReactJSResources/react-webpack-babel"
+  },
+  "author": "ReactJSResources",
+  "license": "MIT",
+  "homepage": "https://github.com/ReactJSResources/react-webpack-babel#readme",
+  "dependencies": {
+    "@babel/cli": "^7.8.3",
```

Comment:
There are a lot of dependencies here, that are dev dependencies actually. Of course, the final bundle will not have these packages, but it's important for the next developer to understand what is dev stack used in the project.

Reply:
True, I just use a boilerplate for the project, if this was for a production project the webpack and the dependencies had to be done by hand removing all the crap.